misc/test/aws_test.go

```
@@ -346,6 +354,9 @@ func TestAccAwsTsEc2Provisioners(t *testing.T) {
 		KeyName: aws.String(keyName),
 	})
 	assert.NoError(t, err)
+	if err != nil {
```

This is just so when running locally it fails faster if AWS creds are borked.